tests/inference/MethodInvocation.java

```diff
@@ -1,13 +1,14 @@
 import universe.qual.*;
 
 class B {
-	//:: fixable-error: (return.type.incompatible)
-	@Rep Object foo ( Object o) { return new Object();}
+	// :: fixable-error: (return.type.incompatible)
+	@Rep Object foo (Object o) { return new Object();}
 }
+
 public class MethodInvocation {
-	@Peer B b = new B();
-	@Rep Object op;
-	//:: fixable-error: (assignment.type.incompatible)
```

Comment:
For a few of these tests I'm wondering why they are moved into a method. For example, this version tests whether the return type is peer or not, because fields are defaulted to peer. Can you add versions of the test that ensure the return type is as expected?

Reply:
Added another test case. 

Regarding the failing test case we had discussed earlier: I found a similar test case in typecheck/topol/Methods.java.